pkg/api/scylla/v1/types_cluster.go

```diff
@@ -331,6 +331,14 @@ type SchedulerTaskSpec struct {
 	// +kubebuilder:default:=3
 	// +optional
 	NumRetries *int64 `json:"numRetries,omitempty"`
+
+	// cron specifies the task schedule as a cron expression. It supports an extended syntax including @monthly, @weekly, @daily, @midnight, @hourly, @every X[h|m|s].
+	// +optional
+	Cron *string `json:"cron,omitempty"`
+
+	// timezone specifies the timezone of cron field.
+	// +optional
+	Timezone *string `json:"timezone,omitempty"`
```

Comment:
please add validation making sure it's set only if the fields that it applies to are present

Reply:
wouldn't it be better to make it transparent, let the client return an error and propagate it to status (globally)?

Reply:
>  the timezone will just be ignored

well, that suck even a bit more, since the user has a timezone set thinking all is well, but it silently runs in a different time then he though

Reply:
done